languages/python/reference/concepts/builtin_types/list.md

```diff
@@ -1,9 +1,19 @@
 # `list`
 
+Lists are dynamic arrays that can be resized by Python at runtime. They are homogenous, meaning they are able to hold any data type, including other lists,  and can even contain themselves (recursive data structures are relatively simple in Python).
```

Comment:
>Lists are dynamic arrays that can be resized by Python at runtime. They are homogenous, meaning they are able to hold any data type, including other lists,  and can even contain themselves (recursive data structures are relatively simple in Python).

I *think* this usage should be heterogeneous - meaning "can contain different data types".  Lists can be both homogenous or heterogeneous ....but you need an `array` to _enforce_  homogeneity.

Reply:
Woops, that was me typing way too late in the evening... I’ll fix that one.